### ISSUE TITLE
check_text: missing text not reported

### ISSUE THREAD

**Issue by g-back** [abaplint/abaplint#885]
```abap
SELECTION-SCREEN PUSHBUTTON 60(30) text-103 USER-COMMAND btn.
```

**Comment by g-back** [comment]
Hm, maybe `TextElement` can be added to `Source`?

Also no reports when used in string templates, ex:
```abap
DATA(html_table) = |{ text-001 }|.
```

**Comment by larshp** [comment]
for
`SELECTION-SCREEN PUSHBUTTON 60(30) text-103 USER-COMMAND btn.`
is it possible to use a variable instead of text-103?

**Comment by g-back** [comment]
Yes, it works the same as in `SELECTION-SCREEN COMMENT`

**Comment by mbtools** [comment]
Related: 
check_syntax error: "b_docu not found" for pushbutton "b_docu"

Test repo show how it works for `selection-screen comment` but errors for `pushbutton`:
https://github.com/mbtools/abapLint-tests

https://github.com/abaplint/abaplint/blob/master/packages/core/src/abap/2_statements/statements/selectionscreen.ts#L58

Change
`  new Source(),`
to
`  opt(alt(new InlineField(), new TextElement())),`
maybe?
